src/content/1.7/themes/reference/templates/notifications.md

```diff
@@ -8,12 +8,12 @@ aliases:
 # Notifications
 
 Throughout the whole front office, the customer can receive notification messages
-from PrestaShop, to inform her about successes or errors for instance.
```

Comment:
i see women getting mad for this. :trollface: can we rephrase it or keep `her / him` maybe?

Reply:
You can actually remove `her` without changing the meaning of the sentence. :)

Reply:
I have pushed the modification. The commit message was hard to find ;)